libburn.yaml

```diff
@@ -0,0 +1,52 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libburn/APKBUILD
+package:
+  name: libburn
```

Comment:
we can instead use single manifest with package name `libburnia` then we can handle with `data.items` pipeline:

```
data:
  - name: components
    items:
      libburn: libburn description
      libisoburn: libisoburn description
      libisofs: libisofs description
```

example here: https://github.com/wolfi-dev/os/blob/b1dc8fb0c9370fb450c1ef4a202fbea7380ba619/cfssl.yaml#L36-L38